runlarky/pom.xml

```diff
@@ -21,10 +21,10 @@
     </repositories>
 
     <properties>
-        <java.version>11</java.version>
+        <java.version>17</java.version>
 
-        <maven.compiler.source>11</maven.compiler.source>
-        <maven.compiler.target>11</maven.compiler.target>
+        <maven.compiler.source>17</maven.compiler.source>
+        <maven.compiler.target>17</maven.compiler.target>
```

Comment:
This feels too easy

Reply:
i don't know why it took the vault team so long to do their java upgrades. this is all you needed to do right?

Reply:
i think it's actually going to work, it more or less worked locally

Reply:
> [ERROR] Failed to execute goal org.apache.maven.plugins:maven-compiler-plugin:3.6.2:compile (default-compile) on project runlarky: Fatal error compiling: error: release version 17 not supported -> [Help 1]

https://app.circleci.com/pipelines/github/verygoodsecurity/starlarky/1294/workflows/5e898454-2c5a-49fd-9d42-d0e172e7c76a/jobs/3571

murphy heard that and disagreed 😂

Reply:
hahahaha that was just because i neglected to update the version of java circleci was installing, the new run seems to be working